### ISSUE TITLE
impact doesn't let me install Optifine with it

### ISSUE THREAD

**Issue by Skygamer10** [ImpactDevelopment/ImpactIssues#3357]
I am trying to use Optifine to disable the explosion effects while crystal pvping, so I tried to install Optifine with impact client (I choose to install Optifine in the installer) and it isn't letting me do so. Is this on my end or is it on the development team.

**Comment by wellsilver** [comment]
Did you download optifine before selecting to install it with impact?

**Comment by Skygamer10** [comment]
No, I did not download optifine before selecting to install it with impact

**Comment by Skygamer10** [comment]
Oh ok. Thank you for letting me know.
